packages/hardhat-cannon/tsconfig.json

```diff
@@ -16,5 +16,6 @@
     "./test",
     "./src",
     "./src/abis/*.json",
+    "types.ts",
```

Comment:
Moved types to the root package folder so they are easier to required when used as dependency.